drivers/storage/ide/atapi/acpi.c

```diff
@@ -0,0 +1,357 @@
+/*
+ * PROJECT:     ReactOS ATA Port Driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     ACPI interface with SATA ports, IDE controllers and drives
+ * COPYRIGHT:   Copyright 2025 Dmitry Borisov <di.sean@protonmail.com>
```

Comment:
You can set copyright years as "2024-2025" on files that you started working on in 2024 🙂